cartography/intel/github/github.py

```diff
@@ -0,0 +1,280 @@
+        })
+        ownerid = f"{github_url}:{repo['owner']['id']}"
+        repo_owners.append({
+            'repoid': githubid,
+            'owner': repo['owner']['login'],
+            'ownerid': ownerid,
+            'type': repo['owner']['__typename'],
+        })
+    results = {
+        'repos': repos,
+        'repo_languages': repo_languages,
+        'repo_owners': repo_owners,
+    }
+    return results
+
+
+def load_github_repos(session, update_tag, repo_data):
+    """
+    Ingest the GitHub repository information
+    :param session: Neo4J session object for server communication
+    :param update_tag: Timestamp used to determine data freshness
+    :param repo_data: repository data objects
+    :return: None
+        """
+    ingest_repo = """
+    UNWIND {RepoData} as repository
+    MERGE (repo:GitHubRepository{id: repository.id})
+    ON CREATE SET repo.firstseen = timestamp(), repo.createdat = repository.createdat
+    set repo.name = repository.name, repo.fullname = repository.fullname,
+    repo.description = repository.description, repo.primarylanguage = repository.primarylanguage,
+    repo.homepage = repository.homepage, repo.defaultbranch = repository.defaultbranch,
+    repo.defaultbranchid = repository.defaultbranchid, repo.private = repository.private,
+    repo.disabled = repository.disabled, repo.archived = repository.archived,
+    repo.locked = repository.locked, repo.giturl = repository.giturl, repo.url = repository.url,
+    repo.sshurl = repository.sshurl, repo.updatedat = repository.updatedat,
+    repo.lastupdated = {UpdateTag}
+    WITH repo
+    MERGE (branch:GitHubBranch{id: repo.defaultbranchid})
+    ON CREATE SET branch.firstseen = timestamp()
+    SET branch.name = repo.defaultbranch, branch.lastupdated = {UpdateTag}
+    WITH repo, branch
+    MERGE (repo)-[r:BRANCH]->(branch)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = r.UpdateTag;"""
+
+    session.run(
+        ingest_repo,
+        RepoData=repo_data,
+        UpdateTag=update_tag,
+    )
+
+
+def load_github_owners(session, update_tag, repo_owners):
+    """
+    Ingest the relationships for repo owners
+    :param session: Neo4J session object for server communication
+    :param update_tag: Timestamp used to determine data freshness
+    :param repo_owners: list of owner to repo mappings
+    :return: None
+    """
+    for owner in repo_owners:
+        ingest_owner = """MERGE (user:#TYPE#{id: {Id}})
+            ON CREATE SET user.firstseen = timestamp()
+            SET user.username = {UserName}, user.lastupdated = {UpdateTag}
+            WITH user
+            MATCH (repo:GitHubRepository{id: {RepoId}})
+            MERGE (user)-[r:OWNER]->(repo)
+            ON CREATE SET r.firstseen = timestamp()
+            SET r.lastupdated = {UpdateTag}"""
+
+        account_type = {'User': "GitHubUser", 'Organization': "GitHubOrganization"}
+
+        session.run(
+            ingest_owner.replace(
```

Comment:
Replace this method of string replacement with string Templating instead.  See https://github.com/lyft/cartography/blob/d595b845c5f9c28f88292e252a0803fa63f1553b/cartography/intel/gcp/crm.py#L179 for an example

Reply:
done